### ISSUE TITLE
ImportError: cannot import name 'utils'

### ISSUE THREAD

**Issue by cedricbonhomme** [suminb/gitstats#6]
It seems that the module utils.py is not copied by setup.py.

```bash
$ pip install --upgrade logbook click gitstats
Requirement already up-to-date: logbook in /home/cedric/.local/share/virtualenvs/freshermeat/lib/python3.6/site-packages
Requirement already up-to-date: click in /home/cedric/.local/share/virtualenvs/freshermeat/lib/python3.6/site-packages
Requirement already up-to-date: gitstats in /home/cedric/.local/share/virtualenvs/freshermeat/lib/python3.6/site-packages

$ which gitstats 
/home/cedric/.local/share/virtualenvs/freshermeat/bin/gitstats

$ gitstats
Traceback (most recent call last):
  File "/home/cedric/.local/share/virtualenvs/freshermeat/bin/gitstats", line 7, in <module>
    from gitstats.__main__ import cli
  File "/home/cedric/.local/share/virtualenvs/freshermeat/lib/python3.6/site-packages/gitstats/__main__.py", line 8, in <module>
    from gitstats.utils import (
ModuleNotFoundError: No module named 'gitstats.utils'

$ python
Python 3.6.3 (default, Dec  6 2017, 11:11:20) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from gitstats import utils
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'utils'
```

Also, it would be nice to add logbook and click as requirements in the file setup.py.

Thanks,

**Comment by suminb** [comment]
Thanks for your feedback. I made some changes and released `0.2.4`, which can be installed via `pip`. Merry Christmas!

**Comment by cedricbonhomme** [comment]
thank you very much! And merry Christmas !
